server/src/avgRating.js

```diff
@@ -0,0 +1,62 @@
+import { fromEvent } from "graphcool-lib";
+
+export default async event => {
+  console.log(event);
+
+  try {
+    //Create Graphcool API (based on https://github.com/graphcool/graphql-request)
+    const graphcool = fromEvent(event);
+    const api = graphcool.api("simple/v1");
+
+    const lockerId = event.data.id;
+    // const avgRating = await getAvgRating(api, lockerId);
```

Comment:
Please remove commented code.

Reply:
These were supposed to be deleted. I don't know how they came back to life.